index.html

```diff
@@ -94,8 +94,6 @@ <h1 class="text-l"><a href="https://amche.in" class="text-white hover:underline"
             style: 'mapbox://styles/planemad/cm3gyibd3004x01qz08rohcsg',
             center: [73.9414, 15.4121],
             zoom: 9.99,
-            pitch: 69.6,
-            bearing: 56,
```

Comment:
@planemad guessing it's intentional to remove default 3d view?

Reply:
Yes, have observed that new users are not aware of how to tilt the map, so its a good idea to remove the map tilt and bearing by default.